torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2479,6 +2525,19 @@ def _make_tensor_helper(shape, low=None, high=None):
                                 dtypes=[torch.bfloat16])),
                    safe_casts_outputs=True,
                    handles_complex_extremals=False),
+    OpInfo('lerp',
+           dtypes=floating_and_complex_types(),
+           # CUDA and ROCM don't support complex inputs
```

Comment:
could you create an issue to add complex support for lerp on CUDA and link it here?